views/components/Graphs/GraphTable.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+  Toolbar,
+  Typography,
+} from '@material-ui/core'
+
+const GraphTable = ({ graph }) => {
+  return (
+    <Paper>
+      <Toolbar>
+        <Typography variant="h6" color="inherit">
+          {graph?.title.toUpperCase()} TABLE
+        </Typography>
+      </Toolbar>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {graph?.labels?.map(({ name }) => (
+              <>
+                <TableCell key={name + 'count'} align="center">
+                  {name}
+                </TableCell>
+                <TableCell key={name + 'percent'} align="center">
+                  {name}%
+                </TableCell>
+              </>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {graph?.dataBySite
+            .slice(0)
+            .reverse()
+            .map((site) => {
+              return (
+                <TableRow hover key={site.name}>
+                  <TableCell align="center">{site.name}</TableCell>
+                  {graph?.labels?.map((fieldLabel) => (
+                    <>
```

Comment:
we need to add a key here.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/175/commits/833d8ffc41e3d610e2ef099e378fb27d1d223c98